src/pages/api/apiUtils/userData.ts

```diff
@@ -1,6 +1,7 @@
 import Cookies from 'cookies';
 import { NextApiResponse } from 'next';
 import { decode } from 'jsonwebtoken';
+import { isArray } from 'lodash';
```

Comment:
We should import lodash modules as `import isArray from 'lodash/isArray'` to avoid importing the entire library